code/game/machinery/harvester.dm

```diff
@@ -124,21 +124,21 @@
 		end_harvesting(success = FALSE)
 		return
 	playsound(src, 'sound/machines/juicer.ogg', 20, TRUE)
-	var/mob/living/carbon/C = occupant
+	var/mob/living/carbon/carbon_occupant = occupant
 	if(!LAZYLEN(operation_order)) //The list is empty, so we're done here
 		end_harvesting(success = TRUE)
 		return
 	var/turf/target = get_step(src, output_dir)
-	for(var/obj/item/bodypart/BP in operation_order) //first we do non-essential limbs
-		BP.drop_limb()
-		C.emote("scream")
-		if(BP.body_zone != "chest")
-			BP.forceMove(target)    //Move the limbs right next to it, except chest, that's a weird one
-			BP.drop_organs()
+	for(var/obj/item/bodypart/limb_to_remove in operation_order) //first we do non-essential limbs
```

Comment:
also is `operation_order` guaranteed to just be a list of only types of `/obj/item/bodypart`? if so, add `as anything`